### ISSUE TITLE
Screen recorder(.gif) doesn't start recording instantly, it has a 1-2 second delay.

### ISSUE THREAD

**Issue by Bodomi** [ShareX/ShareX#2288]
I have made **no** changes to it recently but today it suddenly started doing this.

I go record a gif like I normally do but it doesn't start recording instantly, it takes 1-2 seconds before it stars recording.

Screenshot of my settings: http://i.imgur.com/UuVMkV7.png

Again, I've made no changes at all to the settings, it started doing this today.

**Comment by Bodomi** [comment]
I re-installed the software, now it only has a .5 second delay before it starts recording, still annoying and I would highly prefer no delay like it was before.

**Comment by Bodomi** [comment]
And now it when back to the 1-2 second delay.

Any help?

**Comment by Jaex** [comment]
ShareX runs ffmpeg.exe to record/encode in background. So your anti virus etc. could be delaying opening of executable. There can be many factors for it to delay.

**Comment by Bodomi** [comment]
Ok.

I'm gonna close different programs and see if anything changes.

Thanks for the reply.

**Comment by Bodomi** [comment]
Closed basically everything, except for Norton which I do not believe is the problem, still had the delay.

I decided for the heck of it to to do a full Norton system scan and while that was going on I checked through every program and updated them(AMD Settings driver, Java drivers, Skype etc....) and deleted a few things.
Basically did a mini clean up.

Restarted my computer and now there's no delay.
I will report back if the delay comes back again.

By the way, the reason I do not believe Norton(or basically any program running) is the problem is because it's been working perfectly since I installed it and just yesterday this started and I haven't installed anything new recently or changed anything.
